_schemaV2.graphql

```diff
@@ -8605,6 +8627,9 @@ type Query {
   # List of all artwork attribution classes
   artworkAttributionClasses: [AttributionClass]
 
+  # List of all artwork medium types
+  artworkMediumTypes: [MediumType]
```

Comment:
similar to above, maybe the field is better as `artworkMediums`? (and dropping the -type suffix)

Reply:
I like that!